pkgs/applications/misc/snowsql/linux.nix

```
@@ -0,0 +1,45 @@
+{
+  lib,
+  stdenv,
```

Given that this package isn't built, but rather installs from a pre-built binary use of stdenvNoCC is likely preferable.
```suggestion
  stdenvNoCC,
```

---

I tried this, but run into issues with the `preFixup` step with `patchelf`, as it depends on the dynamic linker which I guess is included with `stdenv`. Do you have any recommendations for solving that?

---

I wasn't aware of that, thanks, @andehen. Unfortunately I don't have any recommendations for that, so how about leaving `stdenv` in then?

---

Yeah, that's probably the eaisest then.